### PR TITLE
Library implementation

### PR DESCRIPTION
**Pull request by MuriloRoque**
In this project, we built a library where you can add books to your collection.

[Live link](https://rawcdn.githack.com/MuriloRoque/library/ecb70030da343ad2a3a0f3d3ed5ab021c1beae09/index.html)

## Requirements

- Keep your code clean.
- Do not commit console.log to your repo. - It’s ugly, it kills performance and it can make confidential data be visible in the browser tools.
- Keep the number of changes/updates to the DOM as low as possible, they are very expensive for the browser.
- Keep the application logic separated from DOM manipulation tasks.
- Keep all the styling logic in CSS files.
If you use node modules 
- lock files should be in the repository to avoid problems with future versions of the dependencies.
If you use ES6 
- use object destructuring to get the values from an object. This way you can avoid repeating a lot of code.

### CODE REVIEW

File: scripts.js
Comment (bellom) [inline scripts.js]: Let's separate the DOM from application logic, you can create another file for DOM.

image

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623451/

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @ TEAM :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 
 
**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

File: scripts.js
Comment (mosaaleb) [inline scripts.js]: - **[OPTIONAL]** [`reset()`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/reset) function is fully supported across all browsers. We can use it to reset all form fields:

```js
document.getElementById('action').reset();
```

File: index.html
Comment (mosaaleb) [inline index.html:13]: - [x] The form should allow the users to mark the book status as read or unread:

**Review by mosaaleb — CHANGES_REQUESTED** [review]
#### Well done @Theophile-Kango @MuriloRoque :clap:

There is still one missing requirements needed to be added before moving on, check out my comments below.

Please add this requirement and submit a new code review.


##### Muhammad Ebeid

**Comment by MuriloRoque** [comment]
Alright, thank you. I wish the first TSE would have noticed these, then we wouldn't have to wait for one more review. [Check last paragraph](https://microverse.zendesk.com/hc/en-us/articles/360026467354-What-do-I-do-if-I-don-t-agree-with-a-code-review-completed-by-a-TSE-). We thought it would be better to leave a button to "Mark as read/unread" than only allowing the user to choose at the creation, that's why we changed it. But since we should do EXACTLY what Odin Project says, we can change it back to the way it was.

**Comment by mosaaleb** [comment]
Hi @MuriloRoque. I am sure that my fellow reviewer didn't notice this requirement and he surely didn't mean to make you wait. Sorry for this inconvenience. 

You can keep the "mark as read/unread" button, it's a nice feature to add. :)

**Comment by bellom** [comment]
> Alright, thank you. I wish the first TSE would have noticed these, then we wouldn't have to wait for one more review. [Check last paragraph](https://microverse.zendesk.com/hc/en-us/articles/360026467354-What-do-I-do-if-I-don-t-agree-with-a-code-review-completed-by-a-TSE-). We thought it would be better to leave a button to "Mark as read/unread" than only allowing the user to choose at the creation, that's why we changed it. But since we should do EXACTLY what Odin Project says, we can change it back to the way it was.

To be candid, I missed that part, so sorry, I could see the features in the app, I guess that made me skip...
Sorry for the inconvenience 🙏

**Review by bellom — APPROVED** [review]
## Hi Team :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**
